COPYING.md

```diff
@@ -0,0 +1,232 @@
+# License Agreement/Contract(s)
+**Severability/salvatorius: If any word, part, portion, provision , of this is or should become illegal, invalid, null or void, rendering this unenforceable in any jurisdiction, that then shall not affect:** 
```

Comment:
Space before comma.

Reply:
Sorted along with the preferred character change